fix(linter): `react/jsx-curly-brace-presence` report for JSXAttributeItem with padding space

Sysix opened the pull request:
This should be fixable, so report it too.
https://github.com/oxc-project/oxc/blob/d94209bfd11e980a826f78d316a7a13f0ccfa8bd/crates/oxc_linter/src/rules/react/jsx_curly_brace_presence.rs#L1149-L1150

camc314 commented:
### Merge activity

* **Aug 8, 10:19 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
fix(linter): `react/jsx-curly-brace-presence` report for JSXAttributeItem with padding space (#12887)

This should be fixable, so report it too.
https://github.com/oxc-project/oxc/blob/d94209bfd11e980a826f78d316a7a13f0ccfa8bd/crates/oxc_linter/src/rules/react/jsx_curly_brace_presence.rs#L1149-L1150
